07.md

```diff
@@ -20,6 +20,9 @@ Aside from these two basic above, the following functions can also be implemente
 async window.nostr.getRelays(): { [url: string]: {read: boolean, write: boolean} } // returns a basic map of relay urls to relay policies
 async window.nostr.nip04.encrypt(pubkey, plaintext): string // returns ciphertext and iv as specified in nip-04
 async window.nostr.nip04.decrypt(pubkey, ciphertext): string // takes ciphertext and iv as specified in nip-04
+async window.nostr.on(eventName: string | symbol, listener: (...args: any[]) => void): EventEmitter //adds the listener function at the end of the listener array for the event named eventName 
```

Comment:
I think the types here are too complicated. What about:

`async window.nostr.on(eventName: "accountChanged", listener: () => void)`

Reply:
Can you please not return EventEmitter? it means everyone has to return an object which fully implements the EventEmitter interface. I think it should have a void value. If they want to add two listeners, just call `window.nostr.on` twice.